common/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "baseUrl": "../",
```

Comment:
Why is the `baseUrl` set to `../` and not `./` is this by mistake?

Reply:
this was indeed a mistake! is now resolved due to the fact that the common module has been removed in favor of using out minio bucket for hosting the assets.